api/mesh/v1alpha1/traffic_route.proto

```diff
@@ -137,15 +137,83 @@ message TrafficRoute {
       map<string, StringMatcher> headers = 3;
     }
 
+    // Modify defines modifications of matched HTTP messages
```

Comment:
Nit: should comments all be complete sentences with a full stop/period?

Reply:
done.